pkg_defs/ska3-core/meta.yaml

```diff
@@ -1,214 +1,269 @@
+---
 package:
   name: ska3-core
-  version: 2020.5
+  version: shiny_2020.12
```

Comment:
Maybe getting pedantic, but what about making this version compliant with standards (https://www.python.org/dev/peps/pep-0440/#local-version-identifiers).  For instance `2020.12+shiny`.

Reply:
done

Reply:
In the business of being pedantic... strictly speaking, this is not the intent of local versions:

> Local version identifiers are used to denote fully API (and, if applicable, ABI) compatible patched versions of upstream projects... The inclusion of the local version label makes it possible to differentiate upstream releases from potentially altered rebuilds by downstream integrators.

Still, I am updating the sanity checks in the CI build so any tag following PEP-0440 can be built, as long as there is some consistency between branch name, release tag name, and release type. This would allow building things like this, abusing the local version.

Reply:
and in the subject... just having a separate name (ska3-flight-shiny or similar) would save us all the abuse (and that is what I believe @jeanconn suggested at some point).